axelrod/tests/unit/test_classification.py

```diff
@@ -7,7 +7,24 @@
 import yaml
 
 import axelrod as axl
-from axelrod.classifier import Classifier, Classifiers, rebuild_classifier_table
+from axelrod.classifier import (
+    Classifier,
+    Classifiers,
+    _Classifiers,
+    memory_depth,
+    rebuild_classifier_table,
+)
+from axelrod.player import Player
+
+
+class TestTitForTat(Player):
```

Comment:
Any objection to:

```suggestion
class TitForTatWithEmptyClassifier(Player):
```

? (This would need to be carried through.)